e2e/test/scenarios/onboarding/setup/setup.cy.spec.ts

```diff
@@ -406,44 +367,129 @@ describeWithSnowplow("scenarios > setup", () => {
     cy.visit(`/setup`);
 
     // 3 - setup/step_seen "welcome"
```

Comment:
I left the comments even though there is the expectGood.... because we also check how many events are dispatched, and the comments are numbered